src/seir/model.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
+import opencor as oc
+
+
+class Model:
+    """
+    SEIR model of Covid-19, as described at
+    https://cpb-ap-se2.wpmucdn.com/blogs.auckland.ac.nz/dist/d/75/files/2017/01/Covid19_SEIR_model.pdf.
+    """
+
+    def __init__(self, sim_duration=300):
```

Comment:
Why is `sim_duration` a parameter to the constructor?

Reply:
Because I forgot to remove it after I made `sim_duration` a parameter to `Model.run()`. My bad. Thanks for spotting it.

Reply:
Fixed in commit 7f4b889.